### PR TITLE
Updated the CCM FAQs page

### PR DESCRIPTION
**Pull request by charanya-j**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md [hunk 1]
```diff
@@ -338,14 +338,13 @@ We continuously update the list of services that work with AutoStopping. Here i
...
-* AutoStopping can scale down the entire task count to zero and start them when new requests come in. While auto-scaling can only scale down the entire active tasks to a minimum task count for that service (min count cannot be zero).
-	+ At scale, the cost of leaving even a single task running per service adds up.
-	+ In the case of the EKS/Kubernetes Cluster, for example, if AutoStopping reduces the number of pods running to zero, auto-scaling will remove that specific node from the cluster. Both can co-exist.
-* AutoStopping allows you to define dependencies between services.
-	+ Dependant services or resources that do not directly receive traffic can also be completely scaled down to zero or shut down based on traffic received at any endpoint, significantly increasing overall cost savings; this is not possible with only native autoscaling. For example, ECS service with an RDS database in the same or different cluster.
+#### How is AutoStopping adding value over Autoscaling?
+
```
Comment (SmritiSatya) [inline docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md:342]: Can we rephrase this? How does AutoStopping add value over Autoscaling?

File: docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md [hunk 2]
```diff
@@ -338,14 +338,13 @@ We continuously update the list of services that work with AutoStopping. Here i
...
+AutoStopping provides several advantages, and it can work alongside Autoscaling to improve efficiency:
+
+* AutoStopping uses real-time traffic as a signal for activity and usage, whereas Autoscaling relies on CPU/memory usage, which may not be an accurate representation of actual usage. 
+* AutoStopping has the ability to scale down the entire task count all the way to zero and back up again when new requests come in, while Autoscaling can only scale down to the minimum task replica count for that service. This means that leaving even a single task running per service can add up at scale, and AutoStopping can help to reduce unnecessary costs.
+* Dependent services or resources that do not directly receive traffic can also be entirely scaled down to zero or shut down based on traffic received at any endpoint, which can significantly increase cost savings overall. For example, an ECS service with an RDS database located in the same or different cluster can be scaled down or shut down entirely based on traffic received, which is not possible with just native Autoscaling.
```
Comment (SmritiSatya) [inline docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md:348]: Dependent services or resources that do not directly receive traffic can also be entirely scaled down to zero or shut down based on traffic received at any endpoint, which can significantly increase overall cost savings. For example, an ECS service with an RDS database located in the same or a different cluster can be scaled down or shut down entirely based on the traffic received, which is not possible with just native Autoscaling.

File: docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md [hunk 3]
```diff
@@ -338,14 +338,13 @@ We continuously update the list of services that work with AutoStopping. Here i
...
-* AutoStopping allows you to define dependencies between services.
-	+ Dependant services or resources that do not directly receive traffic can also be completely scaled down to zero or shut down based on traffic received at any endpoint, significantly increasing overall cost savings; this is not possible with only native autoscaling. For example, ECS service with an RDS database in the same or different cluster.
+#### How is AutoStopping adding value over Autoscaling?
+
+AutoStopping provides several advantages, and it can work alongside Autoscaling to improve efficiency:
+
+* AutoStopping uses real-time traffic as a signal for activity and usage, whereas Autoscaling relies on CPU/memory usage, which may not be an accurate representation of actual usage. 
+* AutoStopping has the ability to scale down the entire task count all the way to zero and back up again when new requests come in, while Autoscaling can only scale down to the minimum task replica count for that service. This means that leaving even a single task running per service can add up at scale, and AutoStopping can help to reduce unnecessary costs.
```
Comment (SmritiSatya) [inline docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md:346]: AutoStopping has the ability to scale down the entire task count all the way to zero and back up again when new requests come in, while Autoscaling can only scale down to the minimum task replica count for that service. This means that leaving even a single task running per service can add up at scale, and AutoStopping can help reduce unnecessary costs.

**Review by SmritiSatya — DISMISSED** [review]

File: docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md (hunk 3 above)
Comment (charanya-j) [reply docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md:346]: Agreed

File: docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md (hunk 2 above)
Comment (charanya-j) [reply docs/frequently-asked-questions/harness-faqs/cloud-cost-management-faqs.md:348]: Agreed

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**